lua/neural.lua

```diff
@@ -34,7 +34,12 @@ function NeuralPromptBorder()
 end
 
 function Neural.setup(options)
-    Config.setup(options)
+    if type(options) == "table" then
+        Config.setup(options)
+    else
+        print('Neural: config options must be a table')
```

Comment:
Consider writing to the error buffer with `vim.api.nvim_err_writeln('[Neural Error]:  ...')`